bahmni-emr-api/src/main/java/org/openmrs/module/bahmniemrapi/laborder/service/LabOrderResultsServiceImpl.java

```diff
@@ -262,6 +262,11 @@ private LabOrderResult createLabOrderResult(EncounterTransaction.Observation obs
         labOrderResult.setAccessionNotes(encounterToAccessionNotesMap.get(orderEncounter.getUuid()));
         labOrderResult.setAction(testOrder.getAction());
         labOrderResult.setOrderUuid(testOrder.getUuid());
+        if(observation.getConcept().getShortName() != null) {
+            labOrderResult.setPreferredTestName(observation.getConcept().getShortName());
+        }else{
+            labOrderResult.setTestName(observation.getConcept().getName());
```

Comment:
@buvaneswari-arun what is the diff between  setPreferredTestName() and setTestName() here ?

Reply:
TestName is the FullySpecifiedName and the preferred name is the shorName - the locale specific name